### PR TITLE
Add covid and surgery forms

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: package/config/configuration/ampathforms/poc_covid19.json [hunk 1]
```diff
@@ -0,0 +1,195 @@
+{
+	"name": "POC Covid",
+	"description" : "Test 1 Description",
+    "version" : "1",
```
Comment (ZacButko) [inline package/config/configuration/ampathforms/poc_covid19.json:4]: Seems there's some weird spacing going on. Is it possible to run this through a code formatter like Prettier?

**Review by ZacButko — APPROVED** [review]
Thanks @reagan-meant

File: package/config/configuration/ampathforms/poc_covid19.json (hunk 1 above)
Comment (reagan-meant) [reply package/config/configuration/ampathforms/poc_covid19.json]: Done

**Review by rbuisson — CHANGES_REQUESTED** [review]

**Comment by rbuisson** [comment]
@reagan-meant  could we get this one done? There is only little changes to be made it seems.

**Review by rbuisson — APPROVED** [review]
Just pasted properly formated JSONs.
Tried locally and this works fine.
Thanks @reagan-meant .

**Pull request merged**